README.md

```diff
@@ -6,185 +6,117 @@ structuring and naming of data (as a set of Interface Data Structures
-```
+You can access the Data Dictionary directly from Python using the `importlib.resources` API:
 
-if you want to generate sphinx documentation on SDCC then you need `Python` and `Saxon`  
+```python
+import importlib.resources
+import xml.etree.ElementTree as ET
 
-For generating IDS Migration guide you need `IMASPy` and `IMAS` should be installed
+# Access the data_dictionary.xml file from the package resources
+xml_path = importlib.resources.files("imas_data_dictionary.resources.schemas") / "data_dictionary.xml"
 
-```bash
-pip install -r docs/requirements.txt
-python generate_docs.py
-pip install .
-```
+# Read the content
+if xml_path.is_file():
+    # Parse the XML content
+    tree = ET.parse(xml_path)
+    root = tree.getroot()
 
-You can access it using following command
+    # Now you can work with the ElementTree
+    # For example, find all IDS elements:
+    for ids in root.findall(".//IDS"):
+        print(f"Found IDS: {ids.get('name')}")
 ```
-$ firefox $PYTHON_PATH/share/doc/imas/sphinx/index.html
-$ dd_doc
-```
-
-### Python scripts
 
-Installer provides `idsinfo` utility which can be used on the command line interface. 
+Alternatively, use the provided `idsinfo` module for higher-level access:
 
-# IDSInfo utility
-The `idsinfo` command provides useful features such as showing
-metadata, listing all variables, searching for text in IDS fields,
-etc.
+```python
+from imas_data_dictionary import idsinfo
 
-## Usage
+# Create object of IDSInfo
+obj_ids_info = idsinfo.IDSInfo()
 
-```sh
-idsinfo -h
-usage: idsinfo [-h] {idspath,metadata,idsnames,search,idsfields,info,doc} ...
+# Get all IDS names
+all_ids = obj_ids_info.get_ids_names()
+print(all_ids)
 
-IDS Info Utilities
+# Get info about a specific IDS
+info = obj_ids_info.get_ids_info("magnetics")
+print(info)
+```
 
-positional arguments:
-  {idspath,metadata,idsnames,search,idsfields,info,doc}
-                        sub-commands help
-    idspath             print ids definition path
-    metadata            print metadata
-    idsnames            print ids names
-    search              Search in ids
-    idsfields           shows all fields from ids
-    info                Query the IDS XML Definition for documentation
-    doc                 Show documentation in the browser
+### Documentation (Sphinx)
 
-options:
-  -h, --help            show this help message and exit
+The documentation is generated by Sphinx and is available [here](https://imas-data-dictionary.readthedocs.io/en/latest/). Note that for generating the `IDS Migration guide` section you will need `imas-python` installed as a prerequisite.
+You can generate and install this documentation with the following steps:
 
+```bash
+pip install -r docs/requirements.txt
+python generate_docs.py
+pip install .
 ```
 
-## Examples:
-
-### Print data dictionary xml path
+After this step you can open the documentation pages via the `dd_doc` command (or the `dd_doclegacy` for the simpler static html version).
```

Comment:
Is  this still true?  It looks from the changes below (removal of scripts from scripts directory) that this may no longer be possible.